website/templates/organization/dashboard/organization_dashboard_base.html

```
@@ -0,0 +1,121 @@
+{% load static %}
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)